Owl/app/src/main/res/values/color.xml

```diff
@@ -13,8 +13,5 @@
   the License.
   -->
 <resources>
-
     <color name="immersive_sys_ui">#33000000</color>
```

Comment:
It looks like this is only used by the theme you're removing in this PR. I think you can also remove this color / entire file?

Reply:
No, it's used as the scrim color in the enableEdgeToEdge method